addons/breathing/functions/fnc_getETCo2.sqf

```diff
@@ -0,0 +1,61 @@
+#include "..\script_component.hpp"
+/*
+ * Author: apo_tle
+ * Calculate patient's ETCo2 level based on status of their vitals. ETCo2 is not its own vital as it does not need to be,
+ * rather I've implemented it as more like a diagnostic tool.
+ *
+ * Arguments:
+ * 0: Patient <OBJECT>
+ *
+ * Return Value:
+ * ETCo2 level <INT>
+ *
+ * Example:
+ * [player] call kat_breathing_fnc_getETCo2;
+ *
+ * Public: No
+ */
+
+ params ["_patient"];
+
+if !(alive _patient) exitWith {0};
+
+_pr = _patient getVariable [QACEGVAR(medical,heartRate), 0];
+_bloodVolume = (_patient getVariable [QACEGVAR(medical,bloodVolume), 6.0]);
+_lostBlood = 6.0 - _bloodVolume; // amount of blood missing from the body
+
+_ptxTarget = _patient getVariable [QGVAR(pneumothorax), 0]; // more deteriorated closer to 4, at 4 likely to become advanced
+_hasHtx = _patient getVariable [QGVAR(hemopneumothorax), false];
+_hasTptx = _patient getVariable [QGVAR(tensopneumothorax), false];
+
+_airwayObstructed = (_patient getVariable [QEGVAR(airway,obstruction), false]);
+_airwayOccluded = (_patient getVariable [QEGVAR(airway,occluded), false]);
+
+//cardiac arrest
+if (_pr == 0) exitWith {0};
+
+//cpr being performed
+if !(_patient getVariable [QACEGVAR(medical,CPR_provider), objNull] isEqualTo objNull) exitWith {
```

Comment:
```suggestion
if (alive (_patient getVariable [QACEGVAR(medical,CPR_provider), objNull])) exitWith {
```

Reply:
No need as we already check the patient is alive earlier in the script

Reply:
I basically used what ACE does to check if there is someone providing CPR in CPRLocal